cluster-install/install.sh

```diff
@@ -1005,27 +1025,31 @@ check_if_image_can_be_pulled_via_curl() {
     if [ -z "$image_repo" ]; then fatal "no image_repo given"; fi
     if [ -z "$image_tag" ]; then fatal "no image_tag given"; fi
 
-    local -a args=('-s' '-w' "\n%{http_code}")
+    local -a args=('-sS' '-w' "\n%{http_code}")
```

Comment:
The `-s` flag makes curl silent, while `-S` makes it output errors to stderr. Combined, it suppresses the normal progress output, but lets errors through.